straxen/get_corrections.py

```diff
@@ -69,7 +67,7 @@ def get_to_pe(run_id, gain_model, n_tpc_pmts):
 
 FIXED_TO_PE = {
     # just some dummy placeholder for nT gains
-    'gain_placeholder': np.repeat(0.001, 494)
```

Comment:
Just a question of taste, but I think 

```
np.ones(494)*0.0085
```

is a bit simpler.

Reply:
My taste goes the other way ;) would only think about replacing the 494 to `straxen.n_tpc_pmts`. (I know I was the one who put it here in the first place, shame on me.)

Reply:
done